update.php

```diff
@@ -55,7 +53,7 @@ function write_log($string) {
   $keys = @parse_ini_file($file, $section);
 // the 'save' switch can be reset by the include file to disallow settings saving
   $save = true;
-  if (isset($_POST['#include'])) include $_POST['#include'];
+  if (isset($_POST['#include'])) include str_replace("//","/","/usr/local/emhttp/{$_POST['#include']}");
```

Comment:
Ronald, it may be safer to use ```realpath("/usr/local/emhttp/{$_POST['#include']}")``` instead of str_replace and then check to make sure the resulting string starts with **/usr/local/emhttp/** to allow the include.

That'll make sure someone doesn't try something sneaky for *#include* like: ../../../etc/sysfile.php